### PR TITLE
[193048] Import people via api

### PR DESCRIPTION
**Pull request by patryk-dabrowski**

### CODE REVIEW

File: backend/hct_mis_api/api/endpoints/rdi/upload_people.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    ImportedHousehold,
+    ImportedIndividual,
+    RegistrationDataImportDatahub,
+)
+
+
+class PeopleSerializer(serializers.ModelSerializer):
+    first_registration_date = serializers.DateTimeField(default=timezone.now)
+    last_registration_date = serializers.DateTimeField(default=timezone.now)
+    household = serializers.ReadOnlyField()
```
Comment (saxix) [inline backend/hct_mis_api/api/endpoints/rdi/upload_people.py:38]: household can be removed. It is an internal design

File: backend/hct_mis_api/apps/household/migrations/0172_migration.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('household', '0171_migration'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='household',
+            name='social_worker',
```
Comment (saxix) [inline backend/hct_mis_api/apps/household/migrations/0172_migration.py:15]: do not use `social_worker`, let add a type field with value "STANDARD" and "SINGLE".
Default value is "STANDARD" use "SINGLE" for any "fake" HH created.

File: backend/hct_mis_api/apps/household/migrations/0172_migration.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
```
Comment (saxix) [inline backend/hct_mis_api/apps/household/migrations/0172_migration.py:18]: this is not needed, we can get it from linked HH

**Review by saxix — CHANGES_REQUESTED** [review]

File: backend/hct_mis_api/api/endpoints/rdi/upload_people.py (hunk 1 above)
Comment (patryk-dabrowski) [reply backend/hct_mis_api/api/endpoints/rdi/upload_people.py]: removed

File: backend/hct_mis_api/apps/household/migrations/0172_migration.py (hunk 2 above)
Comment (patryk-dabrowski) [reply backend/hct_mis_api/apps/household/migrations/0172_migration.py]: changed

**Review by pavlo-mk — APPROVED** [review]

**Pull request merged**